README.md

```diff
@@ -121,6 +122,54 @@ NOTE: Some Swift package manager features (e.g., resources) use rules from [rule
 dependency for `rules_swift_package_manager`. However, you do not need to declare it unless you use
 any of the rules in your project.
 
+### (Optional) Use `swift_package` repository for updating packages
+
+The `swift_deps` module extension will by default generate a `swift_package` repository which can be used to execute `swift package` commands.
+This is useful if you'd like to control the flags and behavior of `swift package`, as well as for using the correct `swift` binary according to the Bazel configured toolchain.
+
+For example, to resolve the `Package.swift` file:
+
+```sh
+bazel run @swift_package//:resolve
```

Comment:
Can you update the `do_test` scripts for the examples to call one of the actions before it builds and executes the tests? The `:resolve` action seems like the best candidate to me.

Reply:
Added a test here